src/containers/edit-profile/notification-tab/NotificationContainer.style.ts

```diff
@@ -0,0 +1,31 @@
+import { TypographyVariantEnum } from '~/types'
+import { rootContainer } from '~/containers/edit-profile/common.style'
+import { switchStyle } from '~/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.styles'
+
+export const styles = {
+  root: {
+    ...rootContainer
+  },
+  titleWithDescription: {
+    title: {
+      typography: TypographyVariantEnum.H6
+    },
+    description: {
+      typography: TypographyVariantEnum.Body2,
+      color: 'primary.500'
+    }
+  },
+  optionsContainer: {
+    padding: '10px 10px 30px'
+  },
+  options: {
+    border: '1px solid',
+    borderColor: 'basic.gray',
+    borderRadius: '5px',
+    padding: '15px',
+    mt: '-1px'
+  },
+  switch: {
+    ...switchStyle
```

Comment:
<img width="1279" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/f0b9ec1c-4819-4d99-9cab-04920ec100e5">

also here. thats smth incorrect

Reply:
the same here
![img2](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110425368/0f19cf56-2d17-489c-9a13-225620ac03db)